integration-tests/src/test/resources/hadoop/wikipedia_hadoop_azure_input_index_task.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
nit: Maybe we can combine the the different cloud input text task json into one by parameterizing the difference in the jobProperties.

Reply:
I thought about doing that, but I was concerned that there might be even more differences in the future and there'd be too many parameters, I think I'll leave this for now